app/src/main/res/values/dimens.xml

```diff
@@ -14,58 +13,44 @@
   <dimen name="elevation_4dp">4dp</dimen>
   <dimen name="elevation_0dp">0dp</dimen>
   <dimen name="padding_296dp">296dp</dimen>
-  <dimen name="padding_72dp">72dp</dimen>
-  <dimen name="padding_24dp">24dp</dimen>
   <dimen name="divider_margin_top">8dp</dimen>
   <dimen name="divider_margin_bottom">8dp</dimen>
   <dimen name="bottom_white_space">160dp</dimen>
-  <dimen name="padding_224">224dp</dimen>
-  <dimen name="padding_128">128dp</dimen>
-  <dimen name="padding_80">80dp</dimen>
-  <dimen name="padding_72">72dp</dimen>
-  <dimen name="padding_64">64dp</dimen>
-  <dimen name="padding_48">48dp</dimen>
-  <dimen name="padding_44">44dp</dimen>
-  <dimen name="padding_40">40dp</dimen>
-  <dimen name="padding_36">36dp</dimen>
-  <dimen name="padding_32">32dp</dimen>
-  <dimen name="padding_24">24dp</dimen>
-  <dimen name="padding_20">20dp</dimen>
-  <dimen name="padding_16">16dp</dimen>
-  <dimen name="padding_12">12dp</dimen>
-  <dimen name="padding_8">8dp</dimen>
-  <dimen name="padding_0">0dp</dimen>
-  <dimen name="padding_28">28dp</dimen>
-  <dimen name="padding_96">96dp</dimen>
-  <dimen name="margin_298">298dp</dimen>
-  <dimen name="margin_208">208dp</dimen>
-  <dimen name="margin_192">192dp</dimen>
-  <dimen name="margin_176">176dp</dimen>
-  <dimen name="margin_170">170dp</dimen>
-  <dimen name="margin_144">144dp</dimen>
-  <dimen name="margin_128">128dp</dimen>
-  <dimen name="margin_124">124dp</dimen>
-  <dimen name="margin_120">120dp</dimen>
-  <dimen name="margin_112">112dp</dimen>
-  <dimen name="margin_84">84dp</dimen>
-  <dimen name="margin_80">80dp</dimen>
-  <dimen name="margin_72">72dp</dimen>
-  <dimen name="margin_68">68dp</dimen>
-  <dimen name="margin_64">64dp</dimen>
-  <dimen name="margin_56">56dp</dimen>
-  <dimen name="margin_48">48dp</dimen>
-  <dimen name="margin_44">44dp</dimen>
-  <dimen name="margin_40">40dp</dimen>
-  <dimen name="margin_36">36dp</dimen>
-  <dimen name="margin_32">32dp</dimen>
-  <dimen name="margin_28">28dp</dimen>
-  <dimen name="margin_24">24dp</dimen>
-  <dimen name="margin_20">20dp</dimen>
-  <dimen name="margin_16">16dp</dimen>
-  <dimen name="margin_12">12dp</dimen>
-  <dimen name="margin_8">8dp</dimen>
-  <dimen name="margin_4">4dp</dimen>
-  <dimen name="margin_0">0dp</dimen>
+  <!-- space  -->
+  <dimen name="space_128dp">128dp</dimen>
+  <dimen name="space_72dp">72dp</dimen>
+  <dimen name="space_64dp">64dp</dimen>
+  <dimen name="space_48dp">48dp</dimen>
+  <dimen name="space_44dp">44dp</dimen>
+  <dimen name="space_40dp">40dp</dimen>
+  <dimen name="space_36dp">36dp</dimen>
+  <dimen name="space_32dp">32dp</dimen>
+  <dimen name="space_28dp">28dp</dimen>
+  <dimen name="space_24dp">24dp</dimen>
+  <dimen name="space_20dp">20dp</dimen>
+  <dimen name="space_16dp">16dp</dimen>
+  <dimen name="space_12dp">12dp</dimen>
+  <dimen name="space_8dp">8dp</dimen>
+  <dimen name="space_0dp">0dp</dimen>
+  <!-- padding -->
+  <dimen name="padding_224dp">224dp</dimen>
```

Comment:
as per the issue description all the dimen values `padding` or `margin`, we have to convert all to `space`. 
No matter if there is `padding_224dp ` and there exist no margin which is 224, we should convert it to save too. 

So, the result of this PR should be:
to convert all the padding and margin to space.

Reply:
Thank a lot @anandwana001  for reviewing my PR.
And I will change all the padding and margin to space, First I thought that I should only change common margin and padding.